lib/cluster.js

```diff
@@ -293,28 +293,46 @@ function masterInit() {
     cluster.emit('setup', settings);
   }
 
-  var debugPortOffset = 1;
+  var debugPortOffset = 0;
+
+  // match[1]: debug argument (e.g. --debug)
+  // match[2]: host part in debug socket (e.g. [::] or 127.0.0.1)
+  // match[3]: debug port (e.g. 5858)
+  var debugArgRegex = /^(--inspect|--debug|--debug-(?:brk|port))(?:=(.+:)?(\d+))?$/;
```

Comment:
Lint error: this line is over 80 chars. Can probably be a const while we're at it:

```js
  const debugArgRegex = 
    /^(--inspect|--debug|--debug-(?:brk|port))(?:=(.+:)?(\d+))?$/;
```